configs/PIMeval_BitSimdV.cfg

```diff
@@ -0,0 +1,8 @@
+num_ranks = 2
+num_bank_per_rank = 128
+num_subarray_per_bank = 32
+num_row_per_subarray = 1024
+num_col_per_subarray = 8192
+simulation_target = PIM_DEVICE_BITSIMD_V
+max_num_threads = 10
```

Comment:
Let's remove this as it will be hardware dependent.

Reply:
This is upper limit. The actual number of threads is also determined by hardware concurrency. 
Could we support comments in this config file, such as "#..."?

Reply:
Does this mean that if hardware thread is 8 it will run using 8 cores and if the hardware thread is 32 it will use 10 cores?

Reply:
Just tested the code and if the hardware has 12 hardware core it is using 10 cores. This is not correct behavior.

Reply:
@fasiddique Thanks for catching this issue. The right logic should be the following function:
num_threads = min(<user input>, std::thread::hardware_concurrency());
Please, correct me if I am wrong.

Reply:
@deyuan, We can add "# ..." as line comment, however DRAMsim3 uses ";" for commenting. My suggestion is to support "#" for line comment and ";" as DRAMsim3 config files. What is you opinion?
[DRAMSim3 config comment example](https://github.com/umd-memsys/DRAMsim3/blob/29817593b3389f1337235d63cac515024ab8fd6e/configs/GDDR5_1Gb_x32.ini#L22C12-L22C50)

Reply:
> @fasiddique Thanks for catching this issue. The right logic should be the following function:
> 
> ```
> num_threads = min(<user input>, std::thread::hardware_concurrency());
> ```
> 
> Please, correct me if I am wrong.

Yes.
But what is the purpose of this parameter? Why aren't we directly taking the hardware thread count? Is this to sweep cores for simulator runtime?